lib/apiservers/engine/backends/system.go

```diff
@@ -170,6 +172,14 @@ func (s *System) SystemInfo() (*types.Info, error) {
 			customInfo := [2]string{systemStatusMemory, units.BytesSize(float64(info.MemTotal))}
 			info.SystemStatus = append(info.SystemStatus, customInfo)
 		}
+		if vchInfo.CPUUsage >= 0 {
+			customInfo := [2]string{systemStatusCPUUsageMhz, fmt.Sprintf("%d Mhz", int(vchInfo.CPUUsage))}
```

Comment:
You should not specify a size here, otherwise you create an array that will be copied almost each time you pass it around:
```
customInfo := []string{systemStatusCPUUsageMhz, fmt.Sprintf("%d Mhz", int(vchInfo.CPUUsage))}
```